index.php

```diff
@@ -38,12 +38,42 @@
 		</div>
 		<?php endif; ?>
 
+
+
+
  		<div class="pull-right visible-desktop home-lab">
 			<a data-toggle="modal" href="#tutorial_video_modal"><img src="img/screencast_thumb.png"></a>
 		</div>
-		<h1><?php echo $homepage_title; ?> <small><?php echo $homepage_subtitle; ?></small></h1>
 
-		<p class="lead">You can use labrador to find and download processed data or request new datasets.
+<h1><img class="pull-left visible-desktop" src=<?php echo $logo; ?> style="margin: 0 20px 10px 0;">
```

Comment:
This should be conditional on `$logo` being set, and also surrounded in quotation marks:

```php
<h1>
<?php if(isset($logo) && !empty($logo)){ ?>
  <img class="pull-left visible-desktop" src="<?php echo $logo; ?>" style="margin: 0 20px 10px 0;">
<?php } ?>
```